jwql/website/apps/jwql/data_containers.py

```diff
@@ -281,18 +308,30 @@ def get_edb_components(request):
 
                 mnemonic_exploration_result.n_rows = len(mnemonic_exploration_result)
 
+                # generate tables for display and download in web app
                 display_table = copy.deepcopy(mnemonic_exploration_result)
+
                 # temporary html file,
                 # see http://docs.astropy.org/en/stable/_modules/astropy/table/
                 tmpdir = tempfile.mkdtemp()
-                path = os.path.join(tmpdir, 'mnemonic_exploration_result_table.html')
-                with open(path, 'w') as tmp:
+                file_name_root = 'mnemonic_exploration_result_table'
+                path_for_html = os.path.join(tmpdir, '{}.html'.format(file_name_root))
+                with open(path_for_html, 'w') as tmp:
                     display_table.write(tmp, format='jsviewer')
-                mnemonic_exploration_result.html_file = path
-                mnemonic_exploration_result.html_file_content = open(path, 'r').read()
-                # pass meta data to have access to number of mnemonics
+                mnemonic_exploration_result.html_file_content = open(path_for_html, 'r').read()
+
+                # pass on meta data to have access to total number of mnemonics
                 mnemonic_exploration_result.meta = meta
 
+                # save file locally to be available for download
+                static_dir = os.path.join(settings.BASE_DIR, 'static')
+                if not os.path.isdir(static_dir):
```

Comment:
Both here and at line 268, this `if` statement can be replace with `utils.ensure_dir_exists()`.

Reply:
done, thanks.